### PR TITLE
Feat/new sharing

### PR DESCRIPTION
**Pull request by GoOz**
- Update sharing discovery UI with wizard style.
- Update sharing permission with UI wizard style and public/avatar instead of Cozy logo.

⚠️ This PR is based on [this PR](https://github.com/cozy/cozy-stack/pull/1841), it will need rebase before merging.

### CODE REVIEW

**Comment by GoOz** [comment]
[(𝐒) ETUO ma page d'acceptation et de choix d'url du partage est personnalisée](https://trello.com/c/v7QGwQGw/1240-%F0%9D%90%92-etuo-ma-page-dacceptation-et-de-choix-durl-du-partage-est-personnalis%C3%A9e)

File: web/auth/auth.go [hunk 1]
```diff
@@ -1201,6 +1202,7 @@ func passphraseRenewForm(c echo.Context) error {
 	}
 	return c.Render(http.StatusOK, "passphrase_renew.html", echo.Map{
 		"Title":                inst.TemplateTitle(),
+		"CozyUI":        				middlewares.CozyUI(inst),
```
Comment (gregorylegarec) [inline web/auth/auth.go]: Tab typo

File: assets/templates/authorize_sharing.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html lang="{{.Locale}}">
   <head>
     <meta charset="utf-8">
-    <title>{{.Title}}</title>
+    <title>Cozy</title>
```
Comment (gregorylegarec) [inline assets/templates/authorize_sharing.html]: Will it be ok with future customizations ?

Comment (GoOz) [reply assets/templates/authorize_sharing.html]: huh? That's odd. I don't remember changing that. I think this is a mistake. 
Good catch, eagle eye ! 👍

**Pull request merged**